cve_bin_tool/cli.py

```diff
@@ -20,34 +20,24 @@
 import pkg_resources
 
 from .cve_scanner import CVEScanner
+from .cvedb import CVEDB, OLD_CACHE_DIR
+from .error_handler import (
+    excepthook,
+    ErrorHandler,
+    InsufficientArgs,
+    EmptyCache,
+    ErrorMode,
+)
 from .input_engine import (
     InputEngine,
     Remarks,
-    InvalidCsvError,
-    InvalidJsonError,
-    MissingFieldsError,
 )
-from .output_engine import OutputEngine
-from .cvedb import CVEDB, OLD_CACHE_DIR, EmptyCache
 from .log import LOGGER
+from .output_engine import OutputEngine
 from .version import VERSION
 from .version_scanner import VersionScanner
 
-ERROR_CODES = {
-    FileNotFoundError: -3,
-    InvalidCsvError: -4,
-    InvalidJsonError: -4,
-    MissingFieldsError: -5,
-}
-
-
-def excepthook(exctype, exc, trace):
-    oldHook(exctype, exc, trace)
-    if ERROR_CODES.get(exctype):
-        sys.exit(ERROR_CODES[exctype])
-
-
-sys.excepthook, oldHook = excepthook, sys.excepthook
+sys.excepthook = excepthook  # Always install excepthook for entrypoint module.
```

Comment:
install our custom excepthook that prints colorized traceback and return exit code.